packages/augur-ui/src/modules/create-market/components/form-details.tsx

```diff
@@ -127,42 +125,41 @@ export default class FormDetails extends React.Component<
           <Subheaders header="Reporting start date and time" subheader="Choose a date and time that is sufficiently after the end of the event. If reporting starts before the event end time the market will likely be reported as invalid. Make sure to factor in potential delays that can impact the event end time. " link />
           <span>
             <DatePicker
-              date={endTimeDropdown ? moment(endTimeDropdown.timestamp * 1000) : null}
+              date={setEndTime ? moment(setEndTime * 1000) : null}
               placeholder="Date"
               displayFormat="MMM D, YYYY"
               id="input-date"
-              onDateChange={(date: Number) => {
+              onDateChange={(date: Moment) => {
                 if (!date) return onChange("setEndTime", "");
-                onChange("setEndTime", formatDate(date.toDate()));
+                onChange("setEndTime", date.startOf('day').unix());
               }}
               isOutsideRange={day =>
                 day.isAfter(moment(currentTimestamp * 1000).add(6, "M")) ||
                 day.isBefore(moment(currentTimestamp * 1000))
               }
               numberOfMonths={1}
               onFocusChange= {({ focused }) => {
-                if (endTime === null) {
-                  const date = moment(currentTimestamp * 1000);
-                  onChange("setEndTime", formatDate(date.toDate()));
+                if (setEndTime === null) {
+                  onChange("setEndTime", currentTimestamp);
                 }
                 this.setState({ dateFocused: focused });
               }}
               focused={s.dateFocused}
-              errorMessage={validations[currentStep].endTime}
+              errorMessage={validations[currentStep].setEndTime}
             />
             <TimeSelector
               hour={hour}
               minute={minute}
               meridiem={meridiem}
-              onChange={(label: string, value: string) => {
+              onChange={(label: string, value: number) => {
                 onChange(label, value)
               }}
               onFocusChange= {(focused: Boolean) => {
                 if (!hour) {
-                  onChange("hour", "12");
+                  onChange("hour", 12);
```

Comment:
clicking on the time selector doesn't fill in the values as supposed to

Reply:
also "0" isn't valid for minutes in the time selector. which makes it a bit messed up